zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -45,17 +58,38 @@ object DbReader {
            update KUBERNETES_CLUSTER set status = "DELETED", destroyedDate = now() where id = $id
            """.update
 
+  def markNodepoolDeleted(id: Long) =
+    sql"""
+           update NODEPOOL set status = "DELETED", destroyedDate = now() where id = $id
+           """.update
+
+  def markRuntimeDeletedQuery(id: Long) =
+    sql"""
+           update CLUSTER set status = "Deleted", destroyedDate = now() where id = $id
+           """.update
+
+  def updateRuntimeStatusQuery(id: Long, status: String) =
+    sql"""
+           update CLUSTER set status = $status where id = $id
+           """.update
+
   def updateNodepoolStatus(id: Long, status: String) =
     sql"""
-           update NODEPOOL set status = $status, destroyedDate = now() where id = $id
+           update NODEPOOL set status = $status where id = $id
```

Comment:
this is the bug fix I mentioned earlier...previously we combined `markXXDeleted` with `updateXXStatus`, but that shouldn't happen because `markXXDeleted` updates `destroyedDate` as well while `updateXXXStatus` shouldnt

Reply:
Yea, I see. 
I wonder if we also need a `markAppError.....()` counterpart so we can update `dateAccessed`?

Reply:
I think of `dateAccessed` is the time when there's external traffic accessing the resource....not whenever the row is being updated

Reply:
When app creation fails, we mark the app's status as `Error`.  Not sure if we wanted to mimic that behaviour here.

Reply:
> When app creation fails, we mark the app's status as Error. Not sure if we wanted to mimic that behaviour here.

what do you mean? mimicing what behavior here?

Reply:
Updating `app.dateAccessed` when updating `app.status` to `Error`

Reply:
oh, so you're saying when we're doing `When app creation fails, we mark the app's status as Error`, we're updating `dateAccessed`? I think we shouldn't do that...I can double check in team channel

Reply:
see conversation in https://broadinstitute.slack.com/archives/GB1VAM14J/p1603223763198200